tests/test_assignment.py

```diff
@@ -5,19 +5,31 @@
 import autograder.assignment
 import autograder.util.invoke
 
+BASE_ERROR_MESSAGE = "Got a False."
```

Comment:
Because of the name, I would just make this a bit more general.
The usage can remain the same, but let the string be something like "Bad result for question."